pkgs/development/python-modules/oracledb/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, cryptography
+, cython
+}:
+
+buildPythonPackage rec {
+  pname = "oracledb";
+  version = "1.2.2";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-3Z9jCE5EZCtISkay/PtPySHzn6z0lKG6sAYo+mQJ9Pw=";
+  };
+
+  buildInputs = [
+    cython
+  ];
+
+  propagatedBuildInputs = [
+    cryptography
+  ];
+
+  doCheck = false;  # Checks need an Oracle database
+
+  pythonImportsCheck = [
+    "oracledb"
+  ];
+
+  meta = with lib; {
+    description = "Python driver for Oracle Database";
+    homepage = "https://oracle.github.io/python-oracledb";
+    changelog = "https://github.com/oracle/python-oracledb/blob/v${version}/doc/src/release_notes.rst";
+    license = with licenses; [ asl20 /* and or */ upl ];
```

Comment:
`and` or `or`?

Reply:
Yes, the package uses a [dual-licensing](https://github.com/oracle/python-oracledb/blob/main/setup.cfg#L18) model. As mentioned https://github.com/NixOS/nixpkgs/pull/217491#discussion_r1112994033.